build.gradle

```diff
@@ -97,6 +97,15 @@ tasks.register('createReleaseTag', CreateGitTag) {
 
 tasks.named('githubRelease') {
     dependsOn('createReleaseTag')
+    notCompatibleWithConfigurationCache("$name task does not support configuration caching")
+}
+
+tasks.named('publishPlugins') {
+    notCompatibleWithConfigurationCache("$name task does not support configuration caching")
+}
+
+tasks.withType(Sign).configureEach {
+    notCompatibleWithConfigurationCache("$name task does not support configuration caching")
```

Comment:
Do you know the plans for making it CC comaptible?

Reply:
I don't, but it appears to be tracked by: [Make `signing` plugin support the configuration cache · Issue #13470 · gradle/gradle](https://github.com/gradle/gradle/issues/13470)